assets/stylesheets/million.scss

```diff
@@ -0,0 +1,111 @@
+// ----- Main Stylesheets ----- //
```

Comment:
Colours aside this is a copypaste of `garnett.scss`. Any non-garnett changes to that stylesheet that were to happen before the campaign will need to be replicated here

After the campaign, there's some minimum health work needed to decommission one of them to avoid this duplication